### PR TITLE
PS-269 (Initial Percona Server 8.0.12 tree)

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
Revert some YaSSL vestiges that managed to sneak in back after the
main YaSSL removal.

Not tested

### CODE REVIEW

File: build-ps/percona-server-8.0_builder.sh [hunk 1]
```diff
@@ -745,17 +741,7 @@ build_tarball(){
     rm -fr ${TARFILE%.tar.gz}
     tar xzf ${TARFILE}
     cd ${TARFILE%.tar.gz}
-    if [ "${YASSL}" = 1 ]; then
-        DIRNAME="tarball_yassl"
-        CMAKE_OPTS="-DWITH_ROCKSDB=1" bash -xe ./build-ps/build-binary.sh --with-jemalloc=../jemalloc/ --with-yassl --with-mecab="${MECAB_INSTALL_DIR}/usr" ../TARGET
-    else
```
Comment (abychko) [inline build-ps/percona-server-8.0_builder.sh:751]: removed both options? YASSL == 1 and YASSL !=1 ?

Comment (laurynas-biveinis) [reply build-ps/percona-server-8.0_builder.sh:751]: Good catch

**Comment by laurynas-biveinis** [comment]
@abychko fixed

**Comment by abychko** [comment]
OK to merge if it works

**Comment by laurynas-biveinis** [comment]
Tested lightly locally

**Pull request merged**